sdcm/cluster.py

```diff
@@ -3857,9 +3859,20 @@ def critical_node_setup_events():
                 current_setup_thread.start()
             while len(setup_results) != len(node_list):
                 verify_node_setup_or_startup(start_time, setup_queue, setup_results)
-            # startup serially
+            # startup
             for node in node_list:
-                node_startup(node, startup_queue)
+                if cl_inst.parallel_startup:
+                    threading.Thread(target=node_startup, args=(node, startup_queue), daemon=True).start()
+                else:
+                    node_startup(node, startup_queue)
```

Comment:
lets try once if we could bootstrap all of the node at the same time

Reply:
yes it also works. Code adapted to be fully parallel and actually also simpler.

Reply:
additionally we need to check that scylla.yaml doesn't include the `force_gossip_topology_changes=true` .  Because in this case raft topology is disabled, and nodes could n't start in parallel. 
Safer way for cluster startup is start first node, check that CONSISTENT_TOPOLOGY_CHANGES is enabled, and then start in parallel all other nodes. 
all next nodes adding to cluster could be added to cluster based on sct parameter or based CONSISTENT_TOPOLOGY_CHANGES got from first node

Reply:
this complicates it a bit. I saw you are implementing this force in different PR - how about giving some more logic in node property (`self.parallel_node_operations = self.params.get("parallel_node_operations") or False` to include this condition when you implement it?

Reply:
More reason why this should be default, it should be carefully set/unset for some cases, i.e. gossip mode ones, upgrade tests can enable it either.

We could start with the tier1 to have it battle proof, and then consider the next steps and what needed to be tested

Reply:
ok, I'll make it false by default and later we can create PR with setting it as true where we can discuss pros and cons of this change.
In the meantime I think I'll refactor this a bit by moving decision part to `node` class instead of `wait_for_init_wrap` so we can add more logic in a clearer way.